lib/samson/form_builder.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+module Samson
+  class FormBuilder < ActionView::Helpers::FormBuilder
+    def input(attribute, as: :text_field, help: false, label: false)
+      content_tag :div, class: 'form-group' do
+        content = label(attribute, label, class: "col-lg-2 control-label")
```

Comment:
when it is 'false' it renders the humanized version of the attribute xy_z -> "Xy Z"